docs/cimpl6LanguageReference.md

```diff
@@ -561,50 +544,54 @@ Cardinality defines the number of repeats that can exist for a `Property`. Cardi
->1. CIMPL allows fixed concept values to be overridden in child classes. Although this seems to violate the notion that constraints should get progressively tighter in subclasses, it is necessary when narrowing the meaning of a class characterized by a concept code. For example, the class _Pneumonia_, with fixed code, _SCT#233604007_, may have a child, _Fungal Pneumonia_, with fixed code _SCT#233613009_, the latter overriding the former. CIMPL will assume, without checking, that the code describing the child class has more constrained semantics than the code it replaces.
+>1. [Bracket notation](#bracket-notation-for-value-choices) must always be used when applying a constraint to a `Value`, if that `Value` has a choice of data types. This makes it clear which of the choices is being constrained.
+>1. CIMPL allows fixed concept values to be overridden in child classes. Although this seems to violate the notion that constraints should get progressively tighter in subclasses, it is necessary when narrowing the meaning of a class characterized by a concept code. For example, the class `Pneumonia`, with fixed code, `SCT#233604007`, may have a child, `Fungal Pneumonia`, with fixed code `SCT#233613009`, the latter overriding the former. CIMPL will assume, without checking, that the code describing the child class has more constrained semantics than the code it replaces.
 >1. If you only need to fix a code in one particular mapping (one version of FHIR, for example), do so using the [`fix`](#fix) keyword in the Map file and not as a constraint in the Class file.
 
 ### Append Constraint
 
-The append `+=` operator appends a fixed concept to a `Property` array with an `Element Value` of `concept`. You can use the `+=` operation repeatedly on the same array, limited only by the cardinality of the array. The append operator can only be applied to a repeating `Property`.
+The append `+=` operator is similar to the fixed value (`=`) operator, but it applies to a repeating `Property` (upper cardinality >1). The `Property` must be an `Element` whose data type is `concept`. The operator appends a fixed `concept` to the array `Property`.  You can use the `+=` operation repeatedly on the same `Property`, only limited by the upper cardinality.
 
-For example, use `+=` to insert a specific _Category_ value into an array of categories.
+Example: Insert two values into an array of categories:
 
-| Example | Syntax |
-|----------|---------|
-| Insert _"Behavior"_ into _Category_ array |  _Category_ `+=` _LNC#54511-1 "Behavior"_ |
-| Add _"Social History"_ to same array | _Category_ `+=` _OBSCAT#social-history "Social History"_ |
+```
+Property:  Category 0..*
+...
+
+Category += LNC#54511-1 "Behavior"
+Category += OBSCAT#social-history "Social History"
+```
 
->**Note:** Appending other data types is not supported, although this feature is planned.
+>**Note:** Appending other data types is currently not supported.
```

Comment:
Suggest removing the word currently.

Reply:
It's a hint that it might change, but I guess we don't have to do that in the reference manual.